### PR TITLE
[Documentation:Submission] Added VCS Gradeable Type

### PR DESCRIPTION
**Pull request by KaelanWillauer**
Updates documentation on https://github.com/Submitty/Submitty/pull/7435

https://submitty.org/student/submission/version_control
Screenshot from 2022-06-03 13-18-54

### CODE REVIEW

**Comment by bmcutler** [comment]
@IDzyre Can you please update the documentation as necessary?
Thanks!

File: _docs/instructor/assignment_preparation/submission_version_control.md [hunk 1]
```diff
@@ -76,6 +76,25 @@ _TODO:_
...
+1. Create a group/organization with a simple name (e.g. cptr101. This part can be skipped if using the instructors repositories.) 
+
+2. Create a project within that group/organization with a simple name (e.g. student101)
+
+3. Clone the repository to your local IDE, and add your assignment(s).
+
+4. Create the course (if not already set up).
+
+5. Create the gradeable, selecting the proper gradeable type, in the required field, 
+type the SSH clone link, but replace your user id with {$user_id} `Should look like git@github.com:{$user_id}/student101.git`
```
Comment (NiaHeermance) [inline _docs/instructor/assignment_preparation/submission_version_control.md:92]: Typo: `with {$user_id}.` (Should have period.)

File: _docs/instructor/assignment_preparation/submission_version_control.md [hunk 2]
```diff
@@ -76,6 +76,25 @@ _TODO:_
...
+
+6. Give your students the URL to the repository, and instruct them to _fork_ it.
+
+__NOTE: Students user ids in Submitty need to be the same as your VCS system__
```
Comment (NiaHeermance) [inline _docs/instructor/assignment_preparation/submission_version_control.md:96]: Probably should have period. `as your VCS system.`

File: _docs/instructor/assignment_preparation/submission_version_control.md [hunk 3]
```diff
@@ -76,6 +76,25 @@ _TODO:_
 _TODO:_
 
 
+### External, Instructor-Created, Student Forked Repositories (e.g Gitlab)
+
+***Submitty will need access to the student's repositories, whether that be adding a SSH key for private repositories, or having public repositories.***
+
+1. Create a group/organization with a simple name (e.g. cptr101. This part can be skipped if using the instructors repositories.)
```
Comment (NiaHeermance) [inline _docs/instructor/assignment_preparation/submission_version_control.md:83]: Typo: I'm assuming you meant `instructor's repositories`?

File: _docs/student/submission/version_control.md [hunk 4]
```diff
@@ -62,6 +62,28 @@ _TODO:_
...
+
+1. Navigate to the course repository specified by your instructor. Duplicate this window, and leave one copy open, as you will come back to it in step 4. 
+
+2. Fork the repository, under your namespace.
+
+3. Clone the repository to your IDE using the 'Clone with SSH' option. Using Github, `git clone git@github.com:myusername/student101.git`.
+
+4. Set the upstream repository using `git remote add upstream SSH_URL`. Replace `SSH_URL` with the 'Clone with SSH' URL from the course repository window from step 1.
+
+4. To pull changes from the upstream repository, use the commands
```
Comment (NiaHeermance) [inline _docs/student/submission/version_control.md:77]: Maybe do `user the commands:` instead, adding a colon.

**Review by NiaHeermance — CHANGES_REQUESTED** [review]
Looks good! Just some small typo stuff to fix and I'll approve it.

**Review by IDzyre — APPROVED** [review]

**Pull request merged**